README.md

```diff
@@ -40,7 +40,7 @@ Some examples are:
     binaries.
 
 ```sh
-docker run -it --net host --pid host --userns host --cap-add audit_control \
```

Comment:
Perhaps the `-it` was added to allow stopping the container with `CTRL-C` (not sure, but it's common)

Reply:
just tested, it can be stopped even without `-it` using `CTRL-C`

Reply:
👍 thanks! saw your PR in my notifications, and thought it was worth mentioning; thanks for checking!